ranger/default.go

```diff
@@ -191,8 +194,10 @@ func defaultResponder(l logger.Logger, url *url.URL, p template.Parser, kr keyri
 		resp.WithErrTemplate(defaultErrTmpl),
 		resp.WithUnauthTemplate(defaultUnauthedTmpl),
 		resp.WithVueTemplate(defaultVueTmpl),
-		resp.WithSessionKey(kr.SessionKey()),
-		resp.WithUserSessionKey(kr.CurrentUserKey()),
+	}
+
+	if len(ctxKeys) > 0 {
```

Comment:
I had imagined this would get kaput as well since we didn't end up using anything outside request/ip/session/current-user?

Reply:
Yeah the use case is in `second-child` and `wall-to-wall` which inject data into `*http.Request.Context` in a middleware to be included when `Vue` is is called for an `Html`.

Reply:
I believe that got removed in the trails refactor from archive so just wall-to-wall which looks like it could just be a middleware and not need the context.

Reply:
Not sure I follow on the suggestion. For `wall-to-wall`, a [middleware injects values](https://github.com/xy-planning-network/wall-to-wall/blob/f8b2c7cb7599846ea9a32f13a865ecf2fc0bd4aa/http/middleware.go#L12-L24) which [`resp.Vue` pulls](https://github.com/xy-planning-network/trails/blob/bd81c167795e335e3149741bf03f7a591ec6dfe2/http/resp/response.go#L350-L355).

Reply:
Now that I'm thinking about it, a simplification would be to add another key to `keys.go` that is used for this special use case and get rid of `CtxKeys`.